<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptoPage</class>
 <widget class="QWidget" name="CryptoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Protect yourself against surveillance by encrypting your Mails with GnuPG. By generating or selecting a public / private key pair now mails can be encrypted using the public key while only you can decrypt them with your private key.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="enableCryptoCheckBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When sending mails KMail automatically looks for encryption keys of your recipients and secures the mail when keys for each recipient can be found.&lt;/p&gt;&lt;p&gt;It also signs your mails cryptographically so that it can be verified that your messages have not been tampered with.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Secure outgoing emails, if possible</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select key for this account:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Kleo::KeySelectionCombo" name="keyCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KNewPasswordWidget" name="passwordWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="wksSupportPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Checking key publishing support ...</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QProgressBar" name="wksCheckProgressBar">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wksPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="wksCheckBox">
         <property name="title">
          <string>Register &amp;this key with your email provider</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="wksDescription">
            <property name="text">
             <string>Your email service provider supports a service to provide your public key so that others can find it.
This means that your public key can be queried from your email provider and others can then use it to secure their email communication with you.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pksPageBox">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="pksCheckBox">
         <property name="title">
          <string>Publish &amp;this key on public key server</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="pksDescription">
            <property name="text">
             <string>Your email service provider does not support key publishing. You should publish your key so that other users can retrieve it and use it to secure their email communication with you.
You can alternatively publish your key on a public key server, but you should be aware that your email address will be publicly listed and they key cannot be removed from there, only revoked.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KNewPasswordWidget</class>
   <extends>QWidget</extends>
   <header>knewpasswordwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Kleo::KeySelectionCombo</class>
   <extends>QComboBox</extends>
   <header location="global">Libkleo/KeySelectionCombo</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
